independent-publisher-connectors/What3Words/apiDefinition.swagger.json

```diff
@@ -0,0 +1,276 @@
+                  "type": "object",
+                  "properties": {
+                    "southwest": {
+                      "type": "object",
+                      "properties": {
+                        "lng": {
+                          "type": "number",
+                          "format": "float",
+                          "title": "Square Longitude",
+                          "description": ""
+                        },
+                        "lat": {
+                          "type": "number",
+                          "format": "float",
+                          "title": "Square Latitude",
+                          "description": ""
+                        }
+                      },
+                      "description": "southwest"
+                    },
+                    "northeast": {
+                      "type": "object",
+                      "properties": {
+                        "lng": {
+                          "type": "number",
+                          "format": "float",
+                          "title": "NE Longitude",
+                          "description": "ne-lng"
+                        },
+                        "lat": {
+                          "type": "number",
+                          "format": "float",
+                          "title": "NE Latitude",
+                          "description": ""
+                        }
+                      },
+                      "description": "northeast"
+                    }
+                  },
+                  "description": "square"
+                },
+                "nearestPlace": {
+                  "type": "string",
+                  "description": "",
+                  "title": "Nearest Place"
+                },
+                "coordinates": {
+                  "type": "object",
+                  "properties": {
+                    "lng": {
+                      "type": "number",
+                      "format": "float",
+                      "description": "",
+                      "title": "Lontitude"
+                    },
+                    "lat": {
+                      "type": "number",
+                      "format": "float",
+                      "description": "",
+                      "title": "Latitide"
+                    }
+                  },
+                  "description": "coordinates"
+                },
+                "words": {
+                  "type": "string",
+                  "description": "",
+                  "title": "Words"
+                },
+                "language": {
+                  "type": "string",
+                  "description": "",
+                  "title": "Language"
+                },
+                "map": {
+                  "type": "string",
+                  "description": "",
+                  "title": "Map"
+                }
+              }
+            }
+          }
+        },
+        "summary": "ConvertToLatLng",
```

Comment:
@mattybeard Please follow the documentation for all the documentation related issues in the swagger. The summary values are expected to be free from grammatical and spelling errors.
Documentation - [Summaries and Descriptions](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)